design/apidsl/metadata.go

```diff
@@ -18,6 +18,11 @@ import (
 //
 //        Metadata("struct:field:name", "MyName")
 //
+// `struct:field:type`: overrides the Go struct field type generated by default by goagen.
+// Applicable to attributes only.
+//
+//        Metadata("struct:field:type", "MyType")
```

Comment:
Please update the comment to reflect the ability to optionally specify a Go import path.